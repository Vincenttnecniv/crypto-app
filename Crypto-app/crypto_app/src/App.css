*,
*::after,
*::before{
    box-sizing: border-box;
    padding: 5px;
    margin: 0px;
}

.Navbar {
  text-align: center;
}

.neo{
  font-size: 60px;
  color: #14a76c;
  margin-top: 50px;

}

body {
  background-color: #28282B;
}

.appCSS {
  color: #14a76c;
  font-family: "Times New Roman"
}

input{
  margin: 10px;
  background-color: #28282B;
}

input[type="submit"]{
  margin: 10px;
  background-color: #00FF00;
  border: none !important;
  color: #28282B;
}

input::placeholder{
  color:white;

  }

  input{
    color: #00FF00;
    font-family: "Times New Roman";
  }

.container{
  display: flex;
  justify-content: center;
  align-items: center;
  margin-top: 100px;
}

input[type="text"]{
  font-family: "Times New Roman"
}

.heading{
  margin-left: 10px;
}

input:hover{
  background-color: black;
  color: #00FF00;
}

input:hover:after{
  color: #28282B;
}

input[type="submit"]:focus{
  background-color: #fff;
  color: #28282B;
}

*{
  font-family: "Times New Roman"
}

/* width */
::-webkit-scrollbar {
  width: 10px;
}

/* Track */
::-webkit-scrollbar-track {
  background: #f1f1f1;
}

/* Handle */
::-webkit-scrollbar-thumb {
  background: lime;
  border-radius: 50px;
}

/* Handle on hover */
::-webkit-scrollbar-thumb:hover {
  background: #555;
  border-radius: 50px;
}

.signIn{
  text-align: center;
  box-shadow: 0px 5px 10px 0px #aaa;  
  background: linear-gradient(to bottom, #BEBEBE 0%, #A8A8A8 20%, #BEBEBE 40%, #909090 60%, #707070 80%, #404040 100%) no-repeat;
  opacity: 0.85;
}

.neon{
  color: #00FF00 !important
}

.output{
  color: #fff !important
}

.home{
 color: snow !important;
 font-size: 60px;
 text-align: center;
}

.home1{
  color: #82fc82 !important;
  font-size: 30px;
  margin-left: 100px;
}

