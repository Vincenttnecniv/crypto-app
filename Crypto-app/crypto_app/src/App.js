import './App.css';
import Home from './components/Home/Home';


const App = () => {

  return (
    <div>
      <div className="App">
        <Home/> {/*Rendering home component*/}
        <br/>
       </div>
    </div>
  );
}

export default App;
