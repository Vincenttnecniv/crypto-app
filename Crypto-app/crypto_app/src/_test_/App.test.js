import {render} from '@testing-library/react';
import App from '../App';
import {BrowserRouter as Router} from 'react-router-dom';
import React from "react";
import renderer from "react-test-renderer";

describe("Testing client or App component", () => { //Client side testing

test('renders app component', async () => { //Unit testing app
  render(
    <Router>
      <App />
    </Router>,
  );
});

  it("App matches DOM Snapshot", () => { //Snapshot testing app
    const tree = renderer.create(
      <Router>
      <App />
      </Router>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});