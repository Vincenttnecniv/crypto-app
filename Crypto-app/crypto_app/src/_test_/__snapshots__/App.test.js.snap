// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing client or App component App matches DOM Snapshot 1`] = `
<div>
  <div
    className="App"
  >
    <div>
      <div
        className="navbar"
      >
        <ul>
          <li>
            <a
              className="active"
              href="/"
              onClick={[Function]}
            >
              HOME
            </a>
          </li>
          <li>
            <a
              className="active"
              href="/register"
              onClick={[Function]}
            >
              REGISTER
            </a>
          </li>
          <li>
            <a
              href="/sign_in"
              onClick={[Function]}
            >
              SIGN IN
            </a>
          </li>
          <li>
            <a
              className="active"
              href="/crypto_currency"
              onClick={[Function]}
            >
              CRYPTO CURRENCY
            </a>
          </li>
          <li>
            <a
              className="active"
              href="/users"
              onClick={[Function]}
            >
              Users
            </a>
          </li>
        </ul>
      </div>
      <br />
      <br />
      <div>
        <br />
        <h1
          className="home"
        >
          How to invest :
        </h1>
        <br />
        <br />
        <br />
        <ol>
          <li
            className="home1"
          >
            Decide how much to invest.
          </li>
          <li
            className="home1"
          >
            Set a goal for your money.
          </li>
          <li
            className="home1"
          >
            Decide on how much help your going to need.
          </li>
          <li
            className="home1"
          >
            Pick & Open an investment account.
          </li>
          <li
            className="home1"
          >
            Choose investments that match your tolerance for risk.
          </li>
          <li
            className="home1"
          >
            Invest in stocks and stock funds.
          </li>
        </ol>
      </div>
      <br />
      <footer>
        <img
          alt="UnloadedImage"
          src="home.jpeg"
          style={
            Object {
              "height": "250px",
              "marginBottom": "-130px",
              "marginLeft": "-44px",
              "marginTop": "50px",
              "width": "106%",
            }
          }
        />
      </footer>
    </div>
     
    <br />
  </div>
</div>
`;
